input/fsh/au-erequesting-servicerequest.fsh

```diff
@@ -189,6 +191,19 @@ Description: "This profile sets minimum expectations for a ServiceRequest resour
 * reasonCode ^extension[http://hl7.org/fhir/StructureDefinition/obligation][4].extension[actor][0].valueCanonical = "http://hl7.org.au/fhir/ereq/ActorDefinition/au-erequesting-actor-patientaccess"
 
 
```

Comment:
Commenting here to get an inline comment. 

Change to remove AU eRequesting ServiceRequest as supported profile from CapabilityStatements is missing.

Reply:
done